### ISSUE TITLE
Security?

### ISSUE THREAD

**Issue by notds** [pbennyjoseph/php-password-uploader#1]
Input is not properly sanitized. 
Is there a defense against uploading other PHP files or revers directory traversal? 

What if I uploaded a crafted file name of ./../evil.php ? 
Granted, probably not executable by default, but you understand.

**Comment by pbennyjoseph** [comment]
Yeah, this could be a problem. I will look into this maybe this weekend
